.github/workflows/updatecli.yml

```
@@ -0,0 +1,48 @@
+name: updatecli
+
+on:
+  workflow_dispatch: ~
+  schedule:
+    - cron: '0 6 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  compose:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
+    steps:
+      - uses: actions/checkout@v4
+
```

do you need to configure `uv` so the updatecli pipeline has the right environment where to run the things on? 

```suggestion
      - name: Install environment
        uses: ./.github/workflows/env-install
```

---

Yes, it will need to be at least installed.